2024-01-22 13:04:15,890 - logs.log - ERROR
Message:"http://localhost:4000/mangadddd" was not found
See if URL was misinputted or if it is currently down 
File: manga.py
Function: fetch_data_async
lineNumber: 21 

2024-01-22 13:04:15,890 - logs.log - WARNING
Message:fetch request to http://localhost:4000/mangadddd returned: None
File: manga.py
Function: main
lineNumber: 42 

2024-01-22 13:04:15,890 - logs.log - ERROR
Message:No data was returned from fetch
File: manga.py
Function: main
lineNumber: 43 

2024-01-22 13:04:34,299 - logs.log - INFO
Message:"http://localhost:4000/manga" fetch successful
File: manga.py
Function: fetch_data_async
lineNumber: 17 

2024-01-22 13:04:34,299 - logs.log - INFO
Message:Fetched data: [{"title":["One Piece","Chainsaw man","Tomadachi game","Tokyo Ghoul","My dress up darling","Black Clover","More than a married couple but not lovers","Gleipnir"]}]
File: manga.py
Function: main
lineNumber: 40 

